### PR TITLE
testing

### PR DESCRIPTION
**Pull request by beaugogh**

### CODE REVIEW

File: src/app/services/cross-table.service.spec.ts [hunk 1]
```diff
@@ -198,134 +198,4 @@ describe('CrossTableService', () => {
     expect(spySet).toHaveBeenCalled();
   });
 
-  it('should correctly create cross constraints for a concept', () => {
```
Comment (gijskant) [inline src/app/services/cross-table.service.spec.ts]: @beaugogh Why did you delete these tests?

File: src/app/services/tree-node.service.ts [hunk 2]
```diff
@@ -185,17 +283,27 @@ export class TreeNodeService {
...
-    const fullName = treeNode['fullName'];
-    let head = fullName.substring(0, fullName.length - tail.length);
-    concept.label = treeNode['name'] + ' (' + head + ')';
-    concept.path = treeNode['conceptPath'];
-    concept.type = <ConceptType> treeNode['type'];
-    concept.code = treeNode['conceptCode'];
-    concept.fullName = treeNode['fullName'];
-    concept.name = treeNode['name'];
-    return concept;
+    if (treeNode['name'] &&
```
Comment (gijskant) [inline src/app/services/tree-node.service.ts]: You can use `treeNode.name` instead.

File: src/app/services/cross-table.service.spec.ts (hunk 1 above)
Comment (gijskant) [reply src/app/services/cross-table.service.spec.ts]: Covered by `cross-table.integration.spec`, right?

Comment (beaugogh) [reply src/app/services/cross-table.service.spec.ts]: yes, should be covered by the integration tests, also, for some reason, the tests failed at these tests frequently, when I removed them, the tests passed

File: src/app/services/tree-node.service.ts (hunk 2 above)
Comment (beaugogh) [reply src/app/services/tree-node.service.ts]: yes, I am actually thinking of refactoring the tree node type, it is something that primeNg provides, which turns out to be more and more ill-fitted for glowing bear, but not priority now
